scripts/paper_evaluations.yml

```diff
@@ -73,6 +73,23 @@ training_checkpoint_sweep:
         - key: passingBehavior
           value: avoid-pass-alive-territory
 
+# Experiment: evaluate adversary vs. KataGo checkpoints.
+katago_ckpt_sweep:
+  num_games_per_matchup: 50
+  # Adversary config
+  adversary_algorithm: AMCTS
+  adversary_visits: 600
+  adversary_path: "/nas/ucb/k8/go-attack/victimplay\
+    /ttseng-avoid-pass-alive-coldstart-39-20221025-175949/models\
+    /t0-s545065216-d136760487/model.bin.gz"  # cyclic-adv-s545m
+  # Victim config
+  victim_visits: 32
+  victim_dir: /nas/ucb/k8/go-attack/katago-networks
+  # We evaluate victims newer or equal to this checkpoint.
+  # We compute victim order using the d value
+  victim_start: kata1-b40c256-s11840935168-d2898845681  # cp505
+  n_victims_per_gpu: 8  # "b40 nets already take 3913 MB." - ttseng
```

Comment:
lol no need to quote me here, just note how you picked this number, or note relevant considerations (e.g., the memory footprint of b40 nets or whatever) for picking this number